dysnix/ai-scale-saver/values.yaml

```diff
@@ -276,16 +282,16 @@ envFromCMs: []
 ## @param envFromSecrets Array of existing Secret names containing env vars
 ##
 envFromSecrets: []
-## @param volumes Array of volumes for the pod(s)
-## @param extraVolumes Optionally specify extra list of additional volumes for the pod(s)
+## @param volumes [array] Array of volumes for the pod(s)
+## @param extraVolumes [array] Optionally specify extra list of additional volumes for the pod(s)
 ##
 volumes:
   - name: configs
     configMap:
       name: '{{ include "common.names.fullname" . }}'
 extraVolumes: []
-## @param volumeMounts Array of volumeMounts for the pod(s) main container
-## @param extraVolumeMounts Optionally specify extra mount list for the pod(s) main container
+## @param volumeMounts [array] Array of volumeMounts for the pod(s) main container
+## @param extraVolumeMounts [array] Optionally specify extra mount list for the pod(s) main container
```

Comment:
Btw, do you suggest we do the same for the base chart ? I mean comment values and specify types?